crates/curp/src/client/stream.rs

```diff
@@ -86,9 +86,12 @@ impl Streaming {
                             );
                             self.state.leader_notifier().listen().await;
                         }
-                        CurpError::ShuttingDown(()) => {
-                            debug!("shutting down stream client background task");
-                            break Err(err);
+                        CurpError::RpcTransport(()) => {
```

Comment:
Another `CurpError::RpcTransport` below. maybe here should be `CurpError::ShuttingDown`?

Reply:
fixed